videos/videos_router.js

```diff
@@ -164,6 +165,7 @@ router.post(
       const videoId = uuid.v4();
       const videoObjectComplete = {
         ...videoObject,
+        video_status: "creating",
         location: `https://artificial-artist.s3.amazonaws.com/${videoId}.mp4`,
         thumbnail: `https://artificial-artist.s3.amazonaws.com/${videoId}.jpg`,
         song_id: song,
```

Comment:
Cool. Got the thumbnails storing in the video table now from DS side. Im relieved we dont have to figure out an FFmpeg install on the Node server to create some thumbnail factory or try to orchestrate one with html5 canvas and ffmpeg on the frontend.